docker/bookkeeper/entrypoint.sh

```diff
@@ -37,16 +37,14 @@ echo "ZK_URL is $ZK_URL"
 echo "BK_DIR is $BK_DIR"
 echo "BK_LEDGERS_PATH is $BK_LEDGERS_PATH"
 
-sed -i 's/3181/'$PORT0'/' ${BK_HOME}/conf/bk_server.conf
-sed -i "s/localhost:2181/${ZK_URL}/" ${BK_HOME}/conf/bk_server.conf
-sed -i 's|journalDirectory=/tmp/bk-txn|journalDirectory='${BK_DIR}'/journal|' ${BK_HOME}/conf/bk_server.conf
-sed -i 's|ledgerDirectories=/tmp/bk-data|ledgerDirectories='${BK_DIR}'/ledgers|' ${BK_HOME}/conf/bk_server.conf
-sed -i 's|indexDirectories=/tmp/data/bk/ledgers|indexDirectories='${BK_DIR}'/index|' ${BK_HOME}/conf/bk_server.conf
-sed -i 's|# zkLedgersRootPath=/ledgers|zkLedgersRootPath='${BK_LEDGERS_PATH}'|' ${BK_HOME}/conf/bk_server.conf
-
-sed -i '/autoRecoveryDaemonEnabled/d' ${BK_HOME}/conf/bk_server.conf
-echo autoRecoveryDaemonEnabled=${BK_AUTORECOVERY} >> ${BK_HOME}/conf/bk_server.conf
-echo useHostNameAsBookieID=${BK_useHostNameAsBookieID} >> ${BK_HOME}/conf/bk_server.conf
+sed -i "s|bookiePort=.*\$|bookiePort=${PORT0}|" ${BK_HOME}/conf/bk_server.conf
+sed -i "s|metadataServiceUri=.*\$|metadataServiceUri=zk://${ZK_URL}${BK_LEDGERS_PATH}|" ${BK_HOME}/conf/bk_server.conf
```

Comment:
do we have system tests with more than 1 zookeeper server ?

Reply:
AFAIK we only use a single ZK endpoint.

Reply:
Does ZK_URL contain ',' or ';' with multiple zk services?